Sonic/Message/MsgActivateLayer.h

```diff
@@ -0,0 +1,18 @@
+﻿#pragma once
+
+#include <Hedgehog/Universe/Engine/hhMessage.h>
+
+namespace Sonic::Message
+{
+	class MsgActivateLayer : public Hedgehog::Universe::MessageTypeSet
+	{
+	public:
+		HH_FND_MSG_MAKE_TYPE(0x0167F0C0);
+		Hedgehog::base::CSharedString m_SetLayerName;
+		bool m_Field14;
+		MsgActivateLayer(const Hedgehog::base::CSharedString& in_SetLayerName, const bool in_Field14) : m_SetLayerName(in_SetLayerName), m_Field14(in_Field14) {}
```

Comment:
base -> Base
in_SetLayerName -> in_rSetLayerName

Reply:
Should be fixed in the latest commit